lib/lf/mesh/entity.h

```diff
@@ -94,7 +102,7 @@ class Entity {
 
   /**
    * @brief Describes the reference element type of this entity.
-   * @return An object of type base::RefEl.
+   * @return An object of type @ref lf::base::RefEl.
```

Comment:
tip: `@ref lf::base::RefEl` is exactly the same as `lf::base::RefEl`. You could even write `base::RefEl` but this would render as `base::RefEl`. The latter works because we are here in the namespace `lf::mesh`;)